src/non-profits.html

```diff
@@ -78,8 +78,39 @@ <h2 class="section-heading">Do you have any questions?</h2>
 <section>
   <h2 class="section-heading">Start your journey here</h2>
   <p>Please fill in the form below to see how we can help bring your project to life.
```

Comment:
I think get rid of it completely to be honest, or get a completely new line. You are right it doesn't sound right at all.